docs/module/lag.md

```diff
@@ -113,6 +113,12 @@ links:
     mlag.peergroup: True # (also) used to derive a unique MAC address for this group of MLAG peers
 ```
 
+### Peerlink configuration
```

Comment:
This left me a bit confused. I think it would be nice to have two short examples, one of them documenting how the peer link can be used as pure routed link (although we know there are other things going on in the background), another one describing how to use VLANs on the peer link (example use case: VRF Lite)

It's also worth mentioning that you chose that netlab configures "allow all VLANs" on the peer link. That is not always the case in real-life deployments.